.github/workflows/integration.yml

```diff
@@ -130,6 +138,39 @@ jobs:
           echo 'generate_password: ${{ inputs.generate_password }}'
           echo 'oidc: ${{ inputs.oidc }}'
           echo 'certificate: ${{ inputs.certificate }}'
+
+  preprocess-scaleway-input:
+    needs: preprocess-input
+    if: ${{ inputs.server_providers == 'scaleway' }}
+    name: Preprocess input variables for scaleway
+    runs-on: ubuntu-latest
+    outputs:
+      kubernetes_versions: ${{ steps.k8s-versions.outputs.versions }}
+    steps:
+      - name: Install scw-cli
+        uses: scaleway/action-scw@v0
+        with:
+          token: ${{ secrets.SCW_ACCESS_KEY }}
+      - id: k8s-versions
+        name: Process k8s version input
+        run: |
+          if [ -z '${{ inputs.kubernetes_versions }}' ]; then
+            echo "Kubernetes versions not specified, determine DO supported versions"
```

Comment:
Kubernetes versions not specified, determine `Scaleway` supported versions

Reply:
good catch, I'll change it